src/Microsoft.DotNet.Interactive.ExtensionLab/MsSqlKernelConnection.cs

```diff
@@ -55,8 +90,8 @@ private async Task InitializeDbContextAsync(MsSqlConnectionOptions options, Kern
             context.Display($"Scaffolding a `DbContext` and initializing an instance of it called `{options.KernelName}` in the C# kernel.", "text/markdown");
 
             var submission1 = @$"
-#r ""nuget:Microsoft.EntityFrameworkCore.Design""
-#r ""nuget:Microsoft.EntityFrameworkCore.SqlServer""
+#r ""nuget:Microsoft.EntityFrameworkCore.Design,3.1.8""
```

Comment:
could we use 3.1.8 as default and have this version part of the #!connect option?

Reply:
That's a good idea. There will likely be other EF options to consider such as an option to pluralize table names. 

Currently, any EF version greater than 3.1.8 won't work due to assumptions on the generated .cs scaffolding code and assembly version conflicts, so we hard code this for now and address the EF issues in another PR.